src/libpriv/rpmostree-rpm-util.c

```diff
@@ -1057,3 +1057,25 @@ rpmostree_fcap_to_xattr_variant (const char *fcap)
                                                    vfsbytes, FALSE));
   return g_variant_ref_sink (g_variant_builder_end (&builder));
 }
+
+gboolean
```

Comment:
How about a comment like:
`/* Return a string representing the checksum we retrieved from the repodata XML, in the form TYPE:VALUE, for example: sha256:0ab34...`